server/src/catalog.rs

```diff
@@ -88,7 +89,8 @@ pub fn new(data_folder: &str) -> Catalog {
                     // HACK: workaround for datafusion 8.0.0 lowercasing table names in queries.
                     let normalized_file_name = file_name.to_ascii_lowercase();
                     if is_dir_entry_a_table(&dir_entry) {
-                        table_metadata.push(new_table_metadata(normalized_file_name, path.to_string()));
+                        table_metadata
+                            .push(new_table_metadata(normalized_file_name, path.to_string()));
                         info!("INFO: initialized table {}", path);
```

Comment:
Manual log level specifier should be removed here as well.

Reply:
Fixed